.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
 
       - name: Set output of cache
         id: yarn-cache
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
```

Comment:
it was not working correctly before? This fixes some issue? Was this tested?

Reply:
> it was not working correctly before? 

Yes it was

> This fixes some issue? 

That syntax is deprecated
<img width="723" alt="Screen Shot 2022-11-02 at 13 55 31" src="https://user-images.githubusercontent.com/43217/199507912-fe245539-18bf-48ad-9e44-6bb4d152198b.png">

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/


> Was this tested?

Yep, I use this elsewhere and nothing changed in the actual behaviour